recipes/ionquant/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set version = "1.10.12" %}
+
+# This key was generated ONLY for the testing of this bioconda package.
+# Users must generate their own key by agreeing to the terms at https://msfragger.arsci.com/ionquant/IonQuant%20Academic%20Use%20License%2005162022.pdf.
+{% set academic_use_only_key = "47cb2953-b4bf7726-2c96f5fe-8f1013aa" %}
+{% set sha256sum = "861a633ab815a34ea54e6e26f318e19b510b25cb8955f3daeb83d6d10c6938ea" %}
+
+package:
+  name: ionquant
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('ionquant', max_pin="x.x") }}
+  script_env:
+    - SHA256SUM={{ sha256sum }}
+
+requirements:
+  host:
+    - python >=3.9
+    - unzip
+  run:
+    - openjdk >=11
+    - python >=3.8
+
+test:
+  commands:
+    - ionquant --key {{ academic_use_only_key }} -Xms512m -Xmx1g --help  2>&1 | grep 'IonQuant'
+
+about:
+  home: https://github.com/Nesvilab/IonQuant
+  license: Academic License (https://msfragger.arsci.com/ionquant/IonQuant%20Academic%20Use%20License%2005162022.pdf)
+  summary: A label free quantification tool
+  description: |
+    IonQuant is a fast and comprehensive tool for MS1 precursor intensity-based quantification for timsTOF PASEF DDA and non-timsTOF (e.g., Orbitrap) data.
+    It enables label-free quantification with false discovery (FDR) controlled match-between-runs (MBR).
+    It can also be used for quantification in labelling-based experiments such as those involving SILAC, dimethyl, or similar labelling strategies.
+
+    IonQuant is available freely for academic research and educational purposes only, in accordance with the terms at https://msfragger.arsci.com/ionquant/IonQuant%20Academic%20Use%20License%2005162022.pdf.
+
+extra:
+  notes: |
```

Want to add yourself as recipe maintainer?

```suggestion
  recipe-maintainers:
    - reid-wagner
  notes: |
```